lib/engaging_networks/action_create_action.rb

```diff
@@ -1,6 +1,6 @@
 module EngagingNetworks
   class ActionCreateAction < ActionModel
```

Comment:
I think we should eliminate this class generally, (or hugely refactor) since the fields defined are somewhat custom per EN client.

Reply:
Internally, we've started using a different approach of having each client have it's own mapping, but probably need to do a bunch of refactoring.